addon/components/bs-datetimepicker.js

```diff
@@ -60,11 +65,64 @@ var bsDateTimePickerComponent = Ember.Component.extend({
 
     this._disabledObserver();
 
+    this.addObserver('disabled', function() {
```

Comment:
Let me think about this further.  One thing I think we can rely on is didReceiveAttrs instead of depending on observers here.